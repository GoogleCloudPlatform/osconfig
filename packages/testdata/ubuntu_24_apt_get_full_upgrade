Inst motd-news-config [13ubuntu10.1] (13ubuntu10.2 Ubuntu:24.04/noble-updates [all])
Conf base-files (13ubuntu10.2 Ubuntu:24.04/noble-updates [amd64])
Conf bsdutils (1:2.39.3-9ubuntu6.2 Ubuntu:24.04/noble-updates [amd64])
Inst perl [5.38.2-3.2build2] (5.38.2-3.2build2.1 Ubuntu:24.04/noble-updates [amd64]) []
Conf perl-base (5.38.2-3.2build2.1 Ubuntu:24.04/noble-updates [amd64]) []
Inst util-linux [2.39.3-9ubuntu6.1] (2.39.3-9ubuntu6.2 Ubuntu:24.04/noble-updates [amd64])
Inst mount [2.39.3-9ubuntu6.1] (2.39.3-9ubuntu6.2 Ubuntu:24.04/noble-updates [amd64])
Inst systemd-dev [255.4-1ubuntu8.4] (255.4-1ubuntu8.6 Ubuntu:24.04/noble-updates [all]) []
Conf libacl1 (2.3.2-1build1.1 Ubuntu:24.04/noble-updates [amd64]) []
Conf libblkid1 (2.39.3-9ubuntu6.2 Ubuntu:24.04/noble-updates [amd64]) []
Inst libkmod2 [31+20240202-2ubuntu7] (31+20240202-2ubuntu7.1 Ubuntu:24.04/noble-updates [amd64]) []
Conf libpcre2-8-0 (10.42-4ubuntu2.1 Ubuntu:24.04/noble-updates [amd64]) []
Conf libselinux1 (3.5-2ubuntu2.1 Ubuntu:24.04/noble-updates [amd64]) []
Inst libsystemd-shared [255.4-1ubuntu8.4] (255.4-1ubuntu8.6 Ubuntu:24.04/noble-updates [amd64]) [systemd:amd64 ]
Conf libsystemd0 (255.4-1ubuntu8.6 Ubuntu:24.04/noble-updates [amd64]) [systemd:amd64 ]
Inst libpam-systemd [255.4-1ubuntu8.4] (255.4-1ubuntu8.6 Ubuntu:24.04/noble-updates [amd64]) [systemd:amd64 ]
Inst udev [255.4-1ubuntu8.4] (255.4-1ubuntu8.6 Ubuntu:24.04/noble-updates [amd64]) []
Conf libudev1 (255.4-1ubuntu8.6 Ubuntu:24.04/noble-updates [amd64])
Conf libaudit-common (1:3.1.2-2.1build1.1 Ubuntu:24.04/noble-updates [all])
Conf libaudit1 (1:3.1.2-2.1build1.1 Ubuntu:24.04/noble-updates [amd64])
Conf libmount1 (2.39.3-9ubuntu6.2 Ubuntu:24.04/noble-updates [amd64])
Inst libuuid1 [2.39.3-9ubuntu6.1] (2.39.3-9ubuntu6.2 Ubuntu:24.04/noble-updates [amd64])
Inst libcryptsetup12 [2:2.7.0-1ubuntu4.1] (2:2.7.0-1ubuntu4.2 Ubuntu:24.04/noble-updates [amd64])
Inst libsmartcols1 [2.39.3-9ubuntu6.1] (2.39.3-9ubuntu6.2 Ubuntu:24.04/noble-updates [amd64])
Inst uuid-runtime [2.39.3-9ubuntu6.1] (2.39.3-9ubuntu6.2 Ubuntu:24.04/noble-updates [amd64])
Inst distro-info-data [0.60ubuntu0.1] (0.60ubuntu0.2 Ubuntu:24.04/noble-updates [all])
Inst python3-problem-report [2.28.1-0ubuntu3.1] (2.28.1-0ubuntu3.5 Ubuntu:24.04/noble-updates [all])
Inst grub2-common [2.12-1ubuntu7] (2.12-1ubuntu7.1 Ubuntu:24.04/noble-updates [amd64]) [grub-pc:amd64 ]
Inst grub-pc-bin [2.12-1ubuntu7] (2.12-1ubuntu7.1 Ubuntu:24.04/noble-updates [amd64]) []
Inst apport-core-dump-handler [2.28.1-0ubuntu3.1] (2.28.1-0ubuntu3.5 Ubuntu:24.04/noble-updates [all])
Inst libopeniscsiusr [2.1.9-3ubuntu5.1] (2.1.9-3ubuntu5.3 Ubuntu:24.04/noble-updates [amd64])
Inst openssh-sftp-server [1:9.6p1-3ubuntu13.8] (1:9.6p1-3ubuntu13.9 Ubuntu:24.04/noble-updates [amd64]) []
Inst openssh-client [1:9.6p1-3ubuntu13.8] (1:9.6p1-3ubuntu13.9 Ubuntu:24.04/noble-updates [amd64])
Inst libattr1 [1:2.5.2-1build1] (1:2.5.2-1build1.1 Ubuntu:24.04/noble-updates [amd64])
Inst libgmp10 [2:6.3.0+dfsg-2ubuntu6] (2:6.3.0+dfsg-2ubuntu6.1 Ubuntu:24.04/noble-updates [amd64])
Inst libgpg-error-l10n [1.47-3build2] (1.47-3build2.1 Ubuntu:24.04/noble-updates [all])
Conf libgpg-error0 (1.47-3build2.1 Ubuntu:24.04/noble-updates [amd64])
Conf libmd0 (1.1.0-2build1.1 Ubuntu:24.04/noble-updates [amd64])
Conf libunistring5 (1.1-2build1.1 Ubuntu:24.04/noble-updates [amd64])
Conf libidn2-0 (2.3.7-2build1.1 Ubuntu:24.04/noble-updates [amd64])
Inst eject [2.39.3-9ubuntu6.1] (2.39.3-9ubuntu6.2 Ubuntu:24.04/noble-updates [amd64])
Inst netplan-generator [1.0.1-1ubuntu2~24.04.1] (1.1.1-1~ubuntu24.04.1 Ubuntu:24.04/noble-updates [amd64]) []
Inst ubuntu-minimal [1.539.1] (1.539.2 Ubuntu:24.04/noble-updates [amd64]) []
Inst initramfs-tools [0.142ubuntu25.2] (0.142ubuntu25.5 Ubuntu:24.04/noble-updates [all]) []
Inst initramfs-tools-bin [0.142ubuntu25.2] (0.142ubuntu25.5 Ubuntu:24.04/noble-updates [amd64]) []
Inst libnetplan1 [1.0.1-1ubuntu2~24.04.1] (1.1.1-1~ubuntu24.04.1 Ubuntu:24.04/noble-updates [amd64])
Inst bsdextrautils [2.39.3-9ubuntu6.1] (2.39.3-9ubuntu6.2 Ubuntu:24.04/noble-updates [amd64])
Inst libdrm-common [2.4.120-2build1] (2.4.122-1~ubuntu0.24.04.1 Ubuntu:24.04/noble-updates [all])
Inst libnghttp2-14 [1.59.0-1ubuntu0.1] (1.59.0-1ubuntu0.2 Ubuntu:24.04/noble-updates [amd64])
Inst libnl-genl-3-200 [3.7.0-0.3build1] (3.7.0-0.3build1.1 Ubuntu:24.04/noble-updates [amd64]) []
Inst libplymouth5 [24.004.60-1ubuntu7] (24.004.60-1ubuntu7.1 Ubuntu:24.04/noble-updates [amd64])
Inst libpackagekit-glib2-18 [1.2.8-2build3] (1.2.8-2ubuntu1.2 Ubuntu:24.04/noble-updates [amd64])
Inst polkitd [124-2ubuntu1] (124-2ubuntu1.24.04.2 Ubuntu:24.04/noble-updates [amd64]) []
Inst libpolkit-gobject-1-0 [124-2ubuntu1] (124-2ubuntu1.24.04.2 Ubuntu:24.04/noble-updates [amd64]) []
Inst plymouth-theme-ubuntu-text [24.004.60-1ubuntu7] (24.004.60-1ubuntu7.1 Ubuntu:24.04/noble-updates [amd64]) []
Inst ubuntu-release-upgrader-core [1:24.04.23] (1:24.04.26 Ubuntu:24.04/noble-updates [all]) []
Inst ubuntu-standard [1.539.1] (1.539.2 Ubuntu:24.04/noble-updates [amd64])
Inst bpftrace [0.20.2-1ubuntu4] (0.20.2-1ubuntu4.3 Ubuntu:24.04/noble-updates [amd64])
Inst cryptsetup-bin [2:2.7.0-1ubuntu4.1] (2:2.7.0-1ubuntu4.2 Ubuntu:24.04/noble-updates [amd64]) []
Inst libdevmapper-event1.02.1 [2:1.02.185-3ubuntu3.1] (2:1.02.185-3ubuntu3.2 Ubuntu:24.04/noble-updates [amd64])
Inst liblvm2cmd2.03 [2.03.16-3ubuntu3.1] (2.03.16-3ubuntu3.2 Ubuntu:24.04/noble-updates [amd64])
Inst fdisk [2.39.3-9ubuntu6.1] (2.39.3-9ubuntu6.2 Ubuntu:24.04/noble-updates [amd64])
Inst grub-efi-amd64-signed [1.202+2.12-1ubuntu7] (1.202.2+2.12-1ubuntu7.1 Ubuntu:24.04/noble-updates [amd64]) []
Inst landscape-common [24.02-0ubuntu5.1] (24.02-0ubuntu5.3 Ubuntu:24.04/noble-updates [amd64])
Inst libldap2 [2.6.7+dfsg-1~exp1ubuntu8] (2.6.7+dfsg-1~exp1ubuntu8.2 Ubuntu:24.04/noble-updates [amd64])
Inst libnvme1t64 [1.8-3build1] (1.8-3ubuntu1 Ubuntu:24.04/noble-updates [amd64])
Inst lvm2 [2.03.16-3ubuntu3.1] (2.03.16-3ubuntu3.2 Ubuntu:24.04/noble-updates [amd64])
Inst python3-dateutil (2.8.2-3ubuntu1 Ubuntu:24.04/noble [all])
Inst python3-botocore (1.34.46+repack-1ubuntu1 Ubuntu:24.04/noble [all])
Conf distro-info-data (0.60ubuntu0.2 Ubuntu:24.04/noble-updates [all])
Conf python3-problem-report (2.28.1-0ubuntu3.5 Ubuntu:24.04/noble-updates [all])
Conf grub2-common (2.12-1ubuntu7.1 Ubuntu:24.04/noble-updates [amd64])
Conf grub-pc-bin (2.12-1ubuntu7.1 Ubuntu:24.04/noble-updates [amd64])
Conf apport-core-dump-handler (2.28.1-0ubuntu3.5 Ubuntu:24.04/noble-updates [all])
Conf libopeniscsiusr (2.1.9-3ubuntu5.3 Ubuntu:24.04/noble-updates [amd64])
Conf openssh-sftp-server (1:9.6p1-3ubuntu13.9 Ubuntu:24.04/noble-updates [amd64])
Conf openssh-client (1:9.6p1-3ubuntu13.9 Ubuntu:24.04/noble-updates [amd64])
Conf libgpg-error-l10n (1.47-3build2.1 Ubuntu:24.04/noble-updates [all])
Conf eject (2.39.3-9ubuntu6.2 Ubuntu:24.04/noble-updates [amd64])
Conf netplan-generator (1.1.1-1~ubuntu24.04.1 Ubuntu:24.04/noble-updates [amd64])
Conf ubuntu-minimal (1.539.2 Ubuntu:24.04/noble-updates [amd64])
