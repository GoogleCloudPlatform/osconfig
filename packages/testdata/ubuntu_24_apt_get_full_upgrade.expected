[]*packages.PkgInfo{
    &packages.PkgInfo{
        Name:    "motd-news-config",
        Arch:    "all",
        RawArch: "",
        Version: "13ubuntu10.2",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "perl",
        Arch:    "x86_64",
        RawArch: "",
        Version: "5.38.2-3.2build2.1",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "util-linux",
        Arch:    "x86_64",
        RawArch: "",
        Version: "2.39.3-9ubuntu6.2",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "mount",
        Arch:    "x86_64",
        RawArch: "",
        Version: "2.39.3-9ubuntu6.2",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "systemd-dev",
        Arch:    "all",
        RawArch: "",
        Version: "255.4-1ubuntu8.6",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "libkmod2",
        Arch:    "x86_64",
        RawArch: "",
        Version: "31+20240202-2ubuntu7.1",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "udev",
        Arch:    "x86_64",
        RawArch: "",
        Version: "255.4-1ubuntu8.6",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "libuuid1",
        Arch:    "x86_64",
        RawArch: "",
        Version: "2.39.3-9ubuntu6.2",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "libcryptsetup12",
        Arch:    "x86_64",
        RawArch: "",
        Version: "2:2.7.0-1ubuntu4.2",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "libsmartcols1",
        Arch:    "x86_64",
        RawArch: "",
        Version: "2.39.3-9ubuntu6.2",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "uuid-runtime",
        Arch:    "x86_64",
        RawArch: "",
        Version: "2.39.3-9ubuntu6.2",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "distro-info-data",
        Arch:    "all",
        RawArch: "",
        Version: "0.60ubuntu0.2",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "python3-problem-report",
        Arch:    "all",
        RawArch: "",
        Version: "2.28.1-0ubuntu3.5",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "grub-pc-bin",
        Arch:    "x86_64",
        RawArch: "",
        Version: "2.12-1ubuntu7.1",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "apport-core-dump-handler",
        Arch:    "all",
        RawArch: "",
        Version: "2.28.1-0ubuntu3.5",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "libopeniscsiusr",
        Arch:    "x86_64",
        RawArch: "",
        Version: "2.1.9-3ubuntu5.3",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "openssh-sftp-server",
        Arch:    "x86_64",
        RawArch: "",
        Version: "1:9.6p1-3ubuntu13.9",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "openssh-client",
        Arch:    "x86_64",
        RawArch: "",
        Version: "1:9.6p1-3ubuntu13.9",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "libattr1",
        Arch:    "x86_64",
        RawArch: "",
        Version: "1:2.5.2-1build1.1",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "libgmp10",
        Arch:    "x86_64",
        RawArch: "",
        Version: "2:6.3.0+dfsg-2ubuntu6.1",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "libgpg-error-l10n",
        Arch:    "all",
        RawArch: "",
        Version: "1.47-3build2.1",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "eject",
        Arch:    "x86_64",
        RawArch: "",
        Version: "2.39.3-9ubuntu6.2",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "netplan-generator",
        Arch:    "x86_64",
        RawArch: "",
        Version: "1.1.1-1~ubuntu24.04.1",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "ubuntu-minimal",
        Arch:    "x86_64",
        RawArch: "",
        Version: "1.539.2",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "initramfs-tools",
        Arch:    "all",
        RawArch: "",
        Version: "0.142ubuntu25.5",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "initramfs-tools-bin",
        Arch:    "x86_64",
        RawArch: "",
        Version: "0.142ubuntu25.5",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "libnetplan1",
        Arch:    "x86_64",
        RawArch: "",
        Version: "1.1.1-1~ubuntu24.04.1",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "bsdextrautils",
        Arch:    "x86_64",
        RawArch: "",
        Version: "2.39.3-9ubuntu6.2",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "libdrm-common",
        Arch:    "all",
        RawArch: "",
        Version: "2.4.122-1~ubuntu0.24.04.1",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "libnghttp2-14",
        Arch:    "x86_64",
        RawArch: "",
        Version: "1.59.0-1ubuntu0.2",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "libnl-genl-3-200",
        Arch:    "x86_64",
        RawArch: "",
        Version: "3.7.0-0.3build1.1",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "libplymouth5",
        Arch:    "x86_64",
        RawArch: "",
        Version: "24.004.60-1ubuntu7.1",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "libpackagekit-glib2-18",
        Arch:    "x86_64",
        RawArch: "",
        Version: "1.2.8-2ubuntu1.2",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "polkitd",
        Arch:    "x86_64",
        RawArch: "",
        Version: "124-2ubuntu1.24.04.2",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "libpolkit-gobject-1-0",
        Arch:    "x86_64",
        RawArch: "",
        Version: "124-2ubuntu1.24.04.2",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "plymouth-theme-ubuntu-text",
        Arch:    "x86_64",
        RawArch: "",
        Version: "24.004.60-1ubuntu7.1",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "ubuntu-release-upgrader-core",
        Arch:    "all",
        RawArch: "",
        Version: "1:24.04.26",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "ubuntu-standard",
        Arch:    "x86_64",
        RawArch: "",
        Version: "1.539.2",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "bpftrace",
        Arch:    "x86_64",
        RawArch: "",
        Version: "0.20.2-1ubuntu4.3",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "cryptsetup-bin",
        Arch:    "x86_64",
        RawArch: "",
        Version: "2:2.7.0-1ubuntu4.2",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "libdevmapper-event1.02.1",
        Arch:    "x86_64",
        RawArch: "",
        Version: "2:1.02.185-3ubuntu3.2",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "liblvm2cmd2.03",
        Arch:    "x86_64",
        RawArch: "",
        Version: "2.03.16-3ubuntu3.2",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "fdisk",
        Arch:    "x86_64",
        RawArch: "",
        Version: "2.39.3-9ubuntu6.2",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "grub-efi-amd64-signed",
        Arch:    "x86_64",
        RawArch: "",
        Version: "1.202.2+2.12-1ubuntu7.1",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "landscape-common",
        Arch:    "x86_64",
        RawArch: "",
        Version: "24.02-0ubuntu5.3",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "libldap2",
        Arch:    "x86_64",
        RawArch: "",
        Version: "2.6.7+dfsg-1~exp1ubuntu8.2",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "libnvme1t64",
        Arch:    "x86_64",
        RawArch: "",
        Version: "1.8-3ubuntu1",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "lvm2",
        Arch:    "x86_64",
        RawArch: "",
        Version: "2.03.16-3ubuntu3.2",
        Source:  packages.Source{},
    },
}